static/js/buddy_data.js

```diff
@@ -120,11 +120,11 @@ export function user_last_seen_time_status(user_id) {
         //
         // We give this vague status for such users; we will get to
         // delete this code when we finish rewriting the presence API.
-        last_seen = $t({defaultMessage: "More than 2 weeks ago"});
+        last_seen = $t({defaultMessage: "more than 2 weeks ago"});
     } else {
         last_seen = timerender.last_seen_status_from_date(last_active_date);
     }
-    return $t({defaultMessage: "Last active: {last_seen}"}, {last_seen});
+    return $t({defaultMessage: "Active {last_seen}"}, {last_seen});
```

Comment:
Is "Active Unknown" an output you're going to get from this? That seems busted.

I think my current thinking is we now likely need the various cases to just return the whole string, rather than this `last_seen` parameter.

Reply:
<img width="630" alt="Skärmavbild 2022-12-21 kl  14 47 22" src="https://user-images.githubusercontent.com/78347729/208920134-f4a223dd-ec87-4a8c-8470-e6121adde5b3.png">

@timabbott is this example a better solution for this comment, and also when the string is translated?

I could not find an example to test with where the user status is unknown, do you know when that happens?

Reply:
No, that doesn't work at all -- the translation library isn't some magic spell that translates things -- it records specific strings for translators to translate, and the `{last_seen}` string does't have any content in another language. Please read up about how internationalization works -- it's a complicated topic, and one that you need to understand in order to work on user-facing strings in any sufficiently successful project. 

In this case, I think you just want each of those clauses to do, for example, `return $t({defaultMessage: "Active more than 2 weeks ago"})` -- with no `last_seen` variable at all.

https://zulip.readthedocs.io/en/latest/translating/internationalization.html

On the question of `Unknown`, I don't know off the top of my head. I'd just modify the code to make it possible it for testing, and/or write a unit test.